src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticIds.cs

```diff
@@ -39,5 +39,6 @@ internal static class RoslynDiagnosticIds
         public const string ImportingConstructorShouldBeObsoleteRuleId = "RS0033";
         public const string ExportedPartsShouldHaveImportingConstructorRuleId = "RS0034";
         public const string RestrictedInternalsVisibleToRuleId = "RS0035";
+        public const string PublicApiFilesMissing = "RS0037";
```

Comment:
I would recommend you to use `RS0036` as yours is a more non-controversial PR which should go in sooner. The other PR should get a merge conflict and move to `RS0037`.